src/duration/extractMinMaxInUnits.js

```diff
@@ -1,5 +1,5 @@
 import durationUnits from './durationUnits';
-import splitSecsToUnits from './splitSecsToUnits';
+import splitSecsToUnits from '../date/splitSecsToUnits';
```

Comment:
you need to know name of an api before before using it anyway, I'll make the change if you're fine